homeassistant/components/cloud/auth_api.py

```diff
@@ -0,0 +1,264 @@
+"""Package to offer tools to authenticate with the cloud."""
+import json
+import logging
+import os
+
+from .const import AUTH_FILE, SERVERS
+from .util import get_mode
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Should have a `REQUIREMENTS` for `botocore`

Reply:
Botocore comes as part of Warrant.